### PR TITLE
 [homeconnect] Use getLinkedChannel rather than getThingChannel ...

### PR DESCRIPTION
**Pull request by lolodomo**
when the purpose is only to call the method updateState

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
Code is based on PR #10955 so we have to wait for the merge of this other PR and then I will rebase this PR.
During this time, the code can already be reviewed.

**Review by bruestel — COMMENTED** [review]
Good idea to get rid of useless updates. Thanks.
As soon as the other PR is merged we can merge this one as well.

**Comment by lolodomo** [comment]
PR is now rebased and ready for a review/merge.

**Review by kaikreuzer — APPROVED** [review]

**Pull request merged**
